conference.js

```diff
@@ -1950,6 +1954,23 @@ export default {
             APP.API.notifySuspendDetected();
         });
 
+        // listens for messages about suspend from power-monitor
+        const transport = new Transport({
```

Comment:
Have you considered adding this code somewhere else? Maybe in the react folder?

Reply:
Yep, but not sure where this can fit, ideas?